patterns/stories-podcast-banner.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Title: Stories Podcast Hero
+ * Slug: twentytwentyfive/stories-podcast-hero
+ * Categories: featured, hero
+ * Keywords: podcast, hero, stories
+ */
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|70","bottom":"var:preset|spacing|70","left":"var:preset|spacing|70","right":"var:preset|spacing|70"}},"color":{"background":"#F6CFF4"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide has-background" style="background-color:#F6CFF4;padding-top:var(--wp--preset--spacing--70);padding-right:var(--wp--preset--spacing--70);padding-bottom:var(--wp--preset--spacing--70);padding-left:var(--wp--preset--spacing--70)"><!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|60","left":"var:preset|spacing|60"}}}} -->
+	<div class="wp-block-columns alignwide"><!-- wp:column {"width":"40%","layout":{"type":"default"}} -->
+		<div class="wp-block-column" style="flex-basis:40%"><!-- wp:image {"className":"wp-block-image size-large is-resized"} -->
```

Comment:
The padding-right of this first column could be at "Regular", so it adjusts better to the designs.